### PR TITLE
Add capi-openstack repo

### PR DESCRIPTION
**Pull request by sbueringer**

### CODE REVIEW

File: infra/gcp/ensure-staging-storage.sh [hunk 1]
```diff
@@ -44,6 +44,7 @@ STAGING_PROJECTS=(
     cip-test
     cluster-api
     cluster-api-aws
+    cluster-api-openstack
```
Comment (detiber) [inline infra/gcp/ensure-staging-storage.sh]: Follow up from separate conversation, I think `capi-openstack` is probably better here, and we can look at porting over renaming `cluster-api-aws` to `capi-aws` as part of v1alpha3 work.

Comment (sbueringer) [reply infra/gcp/ensure-staging-storage.sh]: I shortened the name as discussed

**Comment by dims** [comment]
2019/09/04 15:15:12  ======================= Updates =========================
2019/09/04 15:15:23 Trying to create group: k8s-infra-staging-capi-openstack@kubernetes.io
2019/09/04 15:15:23 > Successfully created group k8s-infra-staging-capi-openstack@kubernetes.io
2019/09/04 15:15:24 > Successfully updated group settings for k8s-infra-staging-capi-openstack@kubernetes.io to allow external members and other security settings
2019/09/04 15:15:25 Added sbueringer@gmail.com to k8s-infra-staging-capi-openstack@kubernetes.io as a MEMBER
2019/09/04 15:15:26 Added jichenjc@cn.ibm.com to k8s-infra-staging-capi-openstack@kubernetes.io as a MEMBER

**Comment by sbueringer** [comment]
> 2019/09/04 15:15:12 ======================= Updates =========================
> 2019/09/04 15:15:23 Trying to create group: [k8s-infra-staging-capi-openstack@kubernetes.io](mailto:k8s-infra-staging-capi-openstack@kubernetes.io)
> 2019/09/04 15:15:23 > Successfully created group [k8s-infra-staging-capi-openstack@kubernetes.io](mailto:k8s-infra-staging-capi-openstack@kubernetes.io)
> 2019/09/04 15:15:24 > Successfully updated group settings for [k8s-infra-staging-capi-openstack@kubernetes.io](mailto:k8s-infra-staging-capi-openstack@kubernetes.io) to allow external members and other security settings
> 2019/09/04 15:15:25 Added [sbueringer@gmail.com](mailto:sbueringer@gmail.com) to [k8s-infra-staging-capi-openstack@kubernetes.io](mailto:k8s-infra-staging-capi-openstack@kubernetes.io) as a MEMBER
> 2019/09/04 15:15:26 Added [jichenjc@cn.ibm.com](mailto:jichenjc@cn.ibm.com) to [k8s-infra-staging-capi-openstack@kubernetes.io](mailto:k8s-infra-staging-capi-openstack@kubernetes.io) as a MEMBER

Thx!

**Comment by jichenjc** [comment]
thanks！

**Pull request merged**